portlets/src/main/webapp/html/profileStats.html

```diff
@@ -0,0 +1,9 @@
+<div class="VuetifyApp">
+    <div id="profile-stats">
+   		<script>
+	   		require(['SHARED/profileStatsBundle'], function(profileStatsApp) {
```

Comment:
please avoid tabulations, this will lead to ugly unreadable and non maintainable files
please configure your IDE to use spaces instead of tabs all time, with tabsize = 2 spaces

Reply:
Done!